simulator/testdata/bad_symlink/symlink.yaml

```diff
@@ -1 +0,0 @@
-foo/bar/bing/bang/boom
```

Comment:
had to remove this, it _really_ didn't like symlinks like this for the upload. Would have to work around.